packages/parser/src/vue-to-tsx.ts

```diff
@@ -40,15 +40,15 @@ const NodeTypes = {
 export const vueToTsx = (code: string) => {
   try {
     const parsed = parse(code)
-    const fileStr = new MagicString(code)
+    const fileStr = new MagicString(`<template>${parsed.descriptor.template?.content}</template>` ?? '')
```

Comment:
@segunadebayo @astahmer 
We could do this instead:
```suggestion
const fileStr = new MagicString(parsed.descriptor.template?.content ?? '')
```
Only did the former to match current tests, but I don't suppose we need the template tag anywhere in the Panda process.

Reply:
I think I added it because otherwise ts-morph had a hard time creating the correct AST nodes
you can check if it's fine here
https://ts-ast-viewer.com/

Reply:
It should stay then